### DISCUSSION TITLE
Resevoir Sampling

### DISCUSSION THREAD

**Discussion by alhardy** [mathnet/mathnet-numerics#798] [Ideas]
Awesome project!

Is there any support with resevoir sampling with histograms? e.g. uniform, exponentially decaying, sliding window, sliding time window

**Comment by gandarez** [comment]
I'm looking at the source code and couldn't find any reference to Resevoir. I think it should be implemented. I may suggest you to post on http://feedback.mathdotnet.com/forums/2060-math-net-numerics

**Comment by cdrnet** [comment]
[Wikipedia: Reservoir sampling](https://en.wikipedia.org/wiki/Reservoir_sampling)
